backend/src/main/java/com/benine/backend/Preset.java

```diff
@@ -0,0 +1,184 @@
+package com.benine.backend;
+
+import org.json.simple.JSONObject;
+
+/**
+ * A moving preset to be able to add to the database.
+ * @author Ege
+ */
+public class Preset {
+
+  private int pan;
+  private int tilt;
+  private int zoom;
+  private int focus;
+  private int iris;
+  private boolean autofocus;
+  private int panspeed;
+  private int tiltspeed;
+  private boolean autoiris;
+
+  /**
+   * Constructs a preset.
+   *
+   * @param pan       The pan of the preset
+   * @param tilt      The tilt of the preset
+   * @param zoom      The zoom of the preset
+   * @param focus     The focus of the prest
+   * @param iris      The iris of the preset
+   * @param autofocus The autofocus of the preset
+   * @param autoiris  The autoiris of the preset
+   * @param tiltspeed The tiltspeed of the preset
+   * @param panspeed  The panspeed of the preset
+   */
+  public Preset(int pan, int tilt, int zoom, int focus, int iris, boolean autofocus, int panspeed, 
+      int tiltspeed, boolean autoiris) {
+    this.pan = pan;
+    this.tilt = tilt;
+    this.zoom = zoom;
+    this.focus = focus;
+    this.iris = iris;
+    this.autofocus = autofocus;
+    this.panspeed = panspeed;
+    this.tiltspeed = tiltspeed;
+    this.autoiris = autoiris;
+  }
+
+  /**
+   * Returns a JSON representation of this object.
+   *
+   * @return JSON representation of this object.
+   */
+  public String toJSON() {
+    JSONObject json = new JSONObject();
+
+    json.put("pan", new Integer(pan));
```

Comment:
Why is a new instance created here? Pan, tilt etc. should already have the correct types.